modules/alloy/src/main/ts/ephox/alloy/api/ui/FloatingToolbarButton.ts

```diff
@@ -26,6 +26,14 @@ import * as Sketcher from './Sketcher';
 import { Toolbar } from './Toolbar';
 import { CompositeSketchFactory } from './UiSketcher';
 
+const shouldSkipFocus = Singleton.value<boolean>();
+
+const toggleWithoutFocusing = (button: AlloyComponent, externals: Record<string, any>) => {
+  shouldSkipFocus.set(true);
```

Comment:
We can try it, but this nearly always ends badly :)

Reply:
I'm out of alternative options here, unfortunately. This seems like the most readable solution. Marking the DOM will provide similar results. It's just DOM instead of the singleton.